conf/modules/lidar_lite.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="lidar_lite" dir="lidar">
+  <doc>
+    <description>Lidar-Lite v1 (silver label) from PulsedLight3D sensor connected over i2c bus</description>
+    <configure name="LIDAR_LITE_I2C_DEV" value="i2c2" description="I2C device to use for Lidar-lite"/>
+    <configure name="LIDAR_LITE_I2C_ADDR" value="0x62" description="slave address of Lidar-lite"/>
+  </doc>
+  <header>
+    <file name="lidar_lite.h"/>
+  </header>
+  <init fun="lidar_lite_init()"/>
+  <periodic fun="lidar_lite_downlink()" freq="10"/> <!-- for debug -->
```

Comment:
a setting will appear in the "modules" settings tab in the GCS (the last tab in settings).